content/post/28-sept-21-oss.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Welcome to OSDC Newsletter!"
+---
+
+<!-- Introduction to Newsletter -->
+
+We will be introducing y'all with the world of free and open source software this week.
+
+# Open Source Explainers
+
+# Open Source Motivation - First hand experiences
```

Comment:
Suggest content related to experienced folks sharing there Open Source journeys

Reply:
https://youtu.be/hwdeUG_gySI

Reply:
https://www.youtube.com/watch?v=4Q7FTjhvZ7Y Project Code Rush - The Beginnings of Netscape / Mozilla Documentary

Reply:
Vue.js - Journey from hobby project to full time open source - https://www.youtube.com/watch?v=OrxmtDw4pVI